projects/helper/solana.js

```diff
@@ -75,9 +83,11 @@ function getAssociatedTokenAddress(mint, owner,  programId = TOKEN_PROGRAM_ID,
 async function getTokenSupplies(tokens, { api } = {}) {
   // const sleepTime = tokens.length > 2000 ? 2000 : 200
   const sleepTime = 200
-  const connection = getConnection()
   tokens = tokens.map(i => typeof i === 'string' ? new PublicKey(i) : i)
-  const res = await runInChunks(tokens, chunk => connection.getMultipleAccountsInfo(chunk), { sleepTime })
+  const res = await withRpcFallback('solana', async (rpc) => {
+    const conn = new Connection(rpc, 'confirmed')
```

Comment:
hmm, this will create a new connection on each call, rpc aggregator support connection?